src/users/tests.py

```diff
@@ -170,6 +172,46 @@ def test_set_mentor_as_active(self):
         self.mentor.refresh_from_db()
         self.assertEqual(self.mentor.active, True)
 
+    def test_patch_major_correct(self):
+    
+        user_params = {
+            'major' : {
+                'name' : 'Test Major'
+            }
+        }
+
+        resp = self.client.patch(
+            self.mentors_update_url,
+            data=user_params,
+            format='json',
+        )
+
+        self.mentor.refresh_from_db()
+        self.assertEqual(self.mentor.major.name, 'Test Major')
+        self.assertEqual(len(Major.objects.all()), 1) 
+        self.mentor2 = factories.MentorFactory()
+        
+    def test_patch_major_404(self):
+
+        self.mentor = factories.MentorFactory(major=self.major)
+        
+        user_params = {
+            'major' : {
+                'name' : 'Wrong'
+            }
+        }
+
+        resp = self.client.patch(
+            self.mentors_update_url,
+            data=user_params,
+            format='json',
+        )
+    
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+        self.mentor.refresh_from_db()
+        self.assertEqual(self.mentor.major.name, 'Test Major')
```

Comment:
i'd also change this to be
`self.assertEqual(self.mentor.major.name, user_params['major']['name'])`